### PR TITLE
feat(DDI-1316): add angular goaChecked directive

### PR DESCRIPTION
**Pull request by chrisolsen**

### CODE REVIEW

File: libs/angular-components/src/lib/checked-directive.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  constructor(protected renderer: Renderer2, protected elementRef: ElementRef) {
+    super(renderer, elementRef);
+  }
+
+  get value(): string {
+    return this._checked ? "checked" : "";
+  }
+
+  set value(checked: any) {
+    console.log("value", !!checked)
```
Comment (vanessatran-ddi) [inline libs/angular-components/src/lib/checked-directive.ts:40]: Remove console.log

**Review by vanessatran-ddi — COMMENTED** [review]
I love this approach to make an angular directive. I have 1 comment to remove console.log.

**Review by ArakTaiRoth — APPROVED** [review]

**Pull request merged**